config/locales/server.en.yml

```diff
@@ -3000,10 +3000,10 @@ en:
     not_available: "Not available. Try %{suggestion}?"
     something_already_taken: "Something went wrong, perhaps the username or email is already registered. Try the forgot password link."
     omniauth_error:
-      generic: "Sorry, there was an error authorizing your account. Please try again."
-      csrf_detected: "Authorization timed out, or you have switched browsers. Please try again."
-      request_error: "An error occurred when starting authorization. Please try again."
-      invalid_iat: "Unable to verify authorization token due to server clock differences. Please try again."
+      generic: "Sorry, there was an error while trying to authenticate with %{provider}. Please try again."
```

Comment:
Should we keep the "authorize" language here vs "authenticate"?

Reply:
Good point, no idea which one is better though 🤔